jme3-examples/src/main/java/jme3test/texture/TestTextureArray.java

```diff
@@ -31,8 +31,8 @@ public void simpleInitApp()
        }
        
        
-       Texture tex1 = assetManager.loadTexture( "Textures/Terrain/Pond/Pond.jpg");
-       Texture tex2 = assetManager.loadTexture("Textures/Terrain/Rock2/rock.jpg");
+       Texture tex1 = assetManager.loadTexture( "Textures/Terrain/Pond/Pond.png");
+       Texture tex2 = assetManager.loadTexture("Textures/Terrain/Rock2/rock.png");
```

Comment:
Why?

Reply:
texture arays not having alpha channel weren't rendered at all, so I added png version of the textures including alpha to be able to run the test